lnwallet/channel.go

```diff
@@ -5695,6 +5695,13 @@ func (lc *LightningChannel) ForceClose() (*LocalForceCloseSummary, error) {
 	// contested state.
 	lc.status = channelDispute
 
+	// Before we return the force close summary, we'll also mark the
+	// channel as borked to indicate that no new state transitions should
+	// be permitted.
+	if err := lc.channelState.MarkBorked(); err != nil {
```

Comment:
should be moved above `lc.status = channelDispute` to prevent in-memory mutations in failure case